test/common/models/Message.js

```diff
@@ -88,10 +88,10 @@ describe('The Message class', function() {
 
     it('should allow defining optional properties through the opts object', function() {
       var message = new jmap.Message({}, 'id', 'threadId', ['inbox'], {
-        from: {
+        from: [{
```

Comment:
You could have put one more Emailer there, currently nothing test that we use your new `nthElementOrDefault` for the `from` field.